src/main/java/com/google/sps/MockData.java

```diff
@@ -0,0 +1,387 @@
+                                            menuUrl,
+                                            businessOrderInfo,
+                                            businessContactUrl,
+                                            linkToBusiness);
+
+    businessName = "Streets of Bangkok";
+    categoryList = Arrays.asList("Mains");
+    businessLat = 1.297275F;
+    businessLong = 103.78742F;
+    businessAddr = "73A Ayer Rajah Crescent, 139957, Singapore";
+    businessLogoUrl = "https://res-1.cloudinary.com/crunchbase-production/image/upload/c_lpad,h_120,w_120,f_auto,b_white,q_auto:eco/v1412921346/nnzir0xhukjpbpmctwy9.jpg";
+    businessPhotoUrlList = Arrays.asList("images/the-oven-bakes/the-oven-bakes-1.PNG","images/the-oven-bakes/the-oven-bakes-2.PNG","images/the-oven-bakes/the-oven-bakes-3.PNG","images/the-oven-bakes/the-oven-bakes-5.PNG");
+
+    businessDescr = "Serving you authentic dishes found along streets of Bangkok\n\n" +
+                    "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut" +
+                    "labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris" +
+                    "nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit" +
+                    "esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in" +
+                    "culpa qui officia deserunt mollit anim id est laborum.";
+
+    menuUrl = "https://www.instagram.com/streetsofbangkok.sg/";
+    businessOrderInfo = "Minimum Order(for delivery): $15\n\n" +
+                                "Islandwide Delivery: $10";
+    businessContactUrl = "https://www.facebook.com/streetsofbangkok/";
+    linkToBusiness = "https://www.instagram.com/streetsofbangkok.sg/";
+
+    Business mockBusiness10 = new Business(businessName, 
+                                            categoryList,
+                                            4.9F, 
+                                            businessLat, 
+                                            businessLong, 
+                                            businessAddr, 
+                                            businessLogoUrl, 
+                                            businessPhotoUrlList,
+                                            businessDescr,
+                                            menuUrl,
+                                            businessOrderInfo,
+                                            businessContactUrl,
+                                            linkToBusiness);
+
+    mockDataList = Arrays.asList(mockBusiness1, mockBusiness2, mockBusiness3, mockBusiness4, mockBusiness5, mockBusiness6, mockBusiness7, mockBusiness8, mockBusiness9, mockBusiness10);
+  }
+
+  /**
+  * Access the mock data through this list.
+  */
+  public List<Business> mockDataList = new ArrayList<Business>();
```

Comment:
I think you should create a private mockDataList at the top of this class like `private List<Business> mockDataList = new ArrayList<Business>();` and then you can have this public getter method `getMockDataList()` where it returns the mockDataList you created.

Reply:
I decided against this because it was going to be a temporary class anyway, so I avoided the getters and setters. Does this mean we avoid having servlets altogether? I was thinking since the call to the datastore was going to be a call to a servlet, it would be easy to mock up the process and change the endpoint (from /mockdata to /businessdatastore for eg)

Reply:
yeah this would mean we can have all storage local but this is not scalable so i think we should stick to Datastore yeah?

i think Jireh's implementation is fine now because we are not keeping this dummy data!